src/services/completions.ts

```diff
@@ -1882,7 +1882,8 @@ namespace ts.Completions {
                         return containingNodeKind === SyntaxKind.ModuleDeclaration;           // module A.|
 
                     case SyntaxKind.OpenBraceToken:
-                        return containingNodeKind === SyntaxKind.ClassDeclaration;            // class A{ |
+                        return containingNodeKind === SyntaxKind.ClassDeclaration             // class A { |
+                            || containingNodeKind === SyntaxKind.ObjectLiteralExpression;     // const obj = { |

```

Comment:
The playground doesn't seem to use commit characters for completions, so it's a bit harder to test, but it looks like this might need to be done for CommaTokens as well

Reply:
Donezo